src/view/point-list-view.js

```diff
@@ -0,0 +1,24 @@
+import {createElement} from '../render.js';
+
+const createPointListTemplate = () => (
+  `<ul class="trip-events__list">
+  </ul>`
+);
```

Comment:
Так как здесь одна строчка то можно так: 
```suggestion
const createPointListTemplate = () => '<ul class="trip-events__list"></ul>';
```
Исправлять необязательно, в текущей записи соблюдается единообразие